### PR TITLE
[tw] Add content/zh-tw/canary-deployment.md

### LINKED ISSUE #2028: [tw] Localize `Canary Deployment` for Traditional Chinese
### Please describe why this term needs to be updated

- Localize `Canary Deployment` in Traditional Chinese
- https://glossary.cncf.io/canary-deployment/

### PR DESCRIPTION
**Pull request by tico88612**
### Describe your changes

Origin content from https://glossary.cncf.io/canary-deployment/

### Related issue number or link (ex: `resolves #issue-number`)

resolves #2028

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Review by hwchiu — APPROVED** [review]

File: content/zh-tw/canary-deployment.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: 金絲雀部署
+status: Completed
+category: 概念
+tags: ["方法論", "應用程式", ""]
+---
+
+## 什麼是金絲雀部署 {#what-it-is}
+
+金絲雀部署是一種部署方法，一開始有兩組環境：一組目前有流量的環境和一組目前無流量且已更新程式碼的環境。
```
Comment (pichuang) [inline content/zh-tw/canary-deployment.md:10]:
```suggestion
金絲雀部署是一種部署策略，一開始有兩組環境：一組目前有即時流量的環境和一組目前無即時流量且已更新程式碼的環境。
```

File: content/zh-tw/canary-deployment.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+流量逐漸從應用程式的原始版本轉移到更新版本。
```
Comment (pichuang) [inline content/zh-tw/canary-deployment.md:11]:
```suggestion
流量會逐漸從應用程式的原始版本轉移到更新後的版本。
```

File: content/zh-tw/canary-deployment.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+它可以從 1% 的流量開始，然後是 10%，25%，以此類推，直到所有流量都轉移到已更新的版本。
+企業可以在正式環境中測試新版本的軟體，得到回饋，錯誤診斷，必要的時候可以快速回退到穩定版本。
```
Comment (pichuang) [inline content/zh-tw/canary-deployment.md:13]:
```suggestion
企業可以在正式環境中測試新版本的軟體，得到回饋、錯誤診斷，並在必要的時候可以快速退回到穩定版本。
```

File: content/zh-tw/canary-deployment.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+「金絲雀」這詞是指「礦坑中的金絲雀」的作法，把金絲雀帶入礦坑中確保礦工們的安全。
+如果出現無味的有害氣體，鳥就會死亡，礦工們知道他們必須儘速離開。
+同樣，如果更新後的程式碼出了問題，流量就會被"疏散"回原始版本中。
```
Comment (pichuang) [inline content/zh-tw/canary-deployment.md:17]:
```suggestion
同樣地，如果更新後的程式碼出了問題，流量就會被"撤離"回原始版本中。
```

File: content/zh-tw/canary-deployment.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+## 解決的問題 {#problem-it-addresses}
+
+無論測試方法有多嚴謹，正式環境中總是會發現一些 bug。
```
Comment (pichuang) [inline content/zh-tw/canary-deployment.md:21]:
```suggestion
無論測試策略有多嚴謹，總是會在正式環境中會發現一些錯誤。
```

File: content/zh-tw/canary-deployment.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+100% 流量從應用程式的一個版本轉移到另一個版本，可能會更高的機率失敗。
```
Comment (pichuang) [inline content/zh-tw/canary-deployment.md:22]:
```suggestion
將 100% 流量從應用程式的一個版本完全切換到另一個版本，可能會導致更嚴重的失敗。
```

File: content/zh-tw/canary-deployment.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+## 如何幫助 {#how-it-helps}
+ 
+金絲雀部署讓企業在大量流量轉移到新版本以前，可以查看新版本軟體在實際使用的情形。
```
Comment (pichuang) [inline content/zh-tw/canary-deployment.md:26]:
```suggestion
金絲雀部署讓企業在大量流量轉移到新版本以前，可以查看新版本軟體在實際場景中使用的情形。
```

**Review by pichuang — APPROVED** [review]
LGTM

File: content/zh-tw/canary-deployment.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+## 解決的問題 {#problem-it-addresses}
+
+無論測試策略有多嚴謹，總是會在正式環境中會發現一些錯誤。
+將 100% 流量從應用程式的一個版本完全切換到另一個版本，可能會導致更嚴重的失敗。
+
+## 如何幫助 {#how-it-helps}
+ 
+金絲雀部署讓企業在大量流量轉移到新版本以前，可以查看新版本軟體在實際場景中使用的情形。
+這種方法讓企業能夠最大限度地減少停機時間，並在新部署出現問題時快速回退上個版本。
+它允許更深度的正式應用測試，而不會對整個使用者體驗產生重大影響。
```
Comment (ydFu) [inline content/zh-tw/canary-deployment.md:28]:
```suggestion
它允許更深度的正式應用程式測試，而不會對整體使用者體驗產生重大影響。
```

File: content/zh-tw/canary-deployment.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
+---
+title: 金絲雀部署
+status: Completed
+category: 概念
+tags: ["方法論", "應用程式", ""]
+---
+
+## 什麼是金絲雀部署 {#what-it-is}
```
Comment (ydFu) [inline content/zh-tw/canary-deployment.md:8]:
```suggestion
## 什麼是金絲雀部署 {#what-it-is}
```
I refer to the content of other zh-tw, it is recommended to be consistent

[1] content/zh-tw/cluster.md 
https://github.com/cncf/glossary/pull/2020/files#diff-db9b1dff7b96f1cf300424e6e661be236510ec7e79fae9b444516d4bc62892b9R8

[2] content/zh-tw/container-orchestration.md 
https://github.com/cncf/glossary/pull/2036/files#diff-262d0ef7001c7fbf22efaf69db9895573f4c02b02f672a93f4b3addc74f5bedbR7

**Pull request merged**